<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hyperspectral_images_app</class>
 <widget class="QMainWindow" name="hyperspectral_images_app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HSNE Volume Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Hierarchical Analysis</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="3">
        <widget class="QTreeWidget" name="_tree">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTabWidget" name="_embeddings_widget">
         <property name="minimumSize">
          <size>
           <width>512</width>
           <height>512</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QPushButton" name="_new_analysis_btn">
           <property name="text">
            <string>Filter and Drill-In</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="_propagate_btn">
           <property name="text">
            <string>Show Area of Influence</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QPushButton" name="_std_view_push_btn">
           <property name="text">
            <string>Standard View</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="_aoi_view_push_btn">
           <property name="text">
            <string>AoI View</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="_view_grp_bx">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="_view_widget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="action_export_images"/>
    <addaction name="action_export_hierarchy"/>
    <addaction name="action_export_influence"/>
    <addaction name="action_export_analysis"/>
   </widget>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_export_images">
   <property name="text">
    <string>Images</string>
   </property>
  </action>
  <action name="action_export_hierarchy">
   <property name="text">
    <string>Hierarchy (.hsne file)</string>
   </property>
  </action>
  <action name="action_export_influence">
   <property name="text">
    <string>Influence of the current selection (.csv file)</string>
   </property>
  </action>
  <action name="action_export_analysis">
   <property name="text">
    <string>Analysis (folder)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
